src/data/finset/lattice.lean

```diff
@@ -531,7 +532,7 @@ lemma sup'_mem
   (s : set α) (w : ∀ x y ∈ s, x ⊔ y ∈ s)
   {ι : Type*} (t : finset ι) (H : t.nonempty) (p : ι → α) (h : ∀ i ∈ t, p i ∈ s) :
   t.sup' H p ∈ s :=
-sup'_induction H p w h
+sup'_induction H p (λ a b ha hb, w a ha b hb) h
```

Comment:
Can we change the order of `sup'_induction` here?

Reply:
yes, done. is it worth changing lean so that things like `\forall {x y \in s}` get turned into `{x : A} (hx : x \in s) ...` instead of the all-implicit default now?

Reply:
I don't think that's a good idea. I'm looking into some more complicated syntax like `\forall ({x y] \in s)` for mathlib4, but I don't think we should make more modifications on the lean 3 side.